modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheAtomicMessageCountSelfTest.java

```diff
@@ -96,7 +96,6 @@ public void testClient() throws Exception {
      * @throws Exception If failed.
      */
     protected void checkMessages(boolean clientMode) throws Exception {
-
         client = clientMode;
 
         startGrids(4);
```

Comment:
May be we can remove 'client' and 'idx' fields and start grids in this way?
```
        if (clientMode)
            startClientGrid(0);

        for (int i = (clientMode ? 1 : 0); i < 4; i++)
            startGrid(i);
```

Reply:
It seems too complicated for me. Let's keep it as is.

Reply:
From other point of view, you are right.
Good. Accepted.